README.md

```diff
@@ -289,32 +289,49 @@ experimenting with Kubernetes.
        cd kind
        make
 
-The Kind environment is a stripped down version without ingress and without identity enabled. So, once pods start up, 
-try using port forwarding to access them. 
+The Kind environment is a stripped down version without ingress and without identity enabled. So, once pods start up,
+try using port forwarding to access them.
 
 For example, try `make port-operate`, and then access operate at localhost: http://localhost:8081
 
 Or, try `make port-tasklist`, and then access task list here: http://localhost:8082
 
+NOTE: learn more about ingress-kind https://kind.sigs.k8s.io/docs/user/ingress/#ingress-nginx
+
+## Run Kind with Ingress and full stack
+
+- Run the make command
+
+`make camunda-ingress chartValues=../../camunda-8-helm-profiles/development/camunda-values.yaml,../../camunda-8-helm-profiles/ingress-kind/camunda-values.yaml`
```

Comment:
This would be nice in the Helm profiles readme to show people how to invoke custom targets and combine values files